.gitignore

```diff
@@ -448,3 +448,11 @@ SkylineTester Results
 /BrowseOnly
 /x64/BrowseOnly
 /qb64.bat
+/pwiz_aux/msrc/utility/vendor_api/Bruker/BAF2SQL-README.txt
```

Comment:
You just mentioned this, but you should not need to alter .gitignore for the project.